contracts/upgradeable_contracts/InterestReceiver.sol

```diff
@@ -0,0 +1,48 @@
+pragma solidity 0.4.24;
+
+import "../interfaces/IChai.sol";
+import "../interfaces/ERC677Receiver.sol";
+import "./Initializable.sol";
+import "./Ownable.sol";
+
```

Comment:
Consider to add a script to deploy this contract and initialize the owner similar to `deploy/src/utils/deployERC20Token.js`

Reply:
Addressed by https://github.com/poanetwork/tokenbridge-contracts/pull/380/commits/9c0c9a2ce003096882558a424f7fc2be84f1de01